src/main/java/frc/robot/Drivetrain.java

```diff
@@ -75,9 +75,11 @@ public void tankDrive(double leftDrive, double rightDrive){
      * motors.
      *
      * @param in the input to check
+     * @param boost whether or not the boost button is being pressed 
      * @return 0.0 if {@code in} is less than abs(DEADBAND_LIMIT) else {@code in}
      */
-    public double deadband(double in) {
-        return Math.abs(in) > DEADBAND_LIMIT ? in : 0.0;
+    public double deadband(double in, boolean boost) {
```

Comment:
This function is actually doing a little too much at this point. It's doing both "deadband" and "boost" functions which should probably be separated into their own functions. Once you do so, `deadband` doesn't need to be in the drivetrain class at all, in fact it might be useful to keep separate so that other classes can use it as well (manual control of the shooter angle, collection system, etc.) 

The boost code itself is pretty hard to follow, I think we should move all instances of `0.6` to a constant (also to make it easy to change later). Doubling up the terenary operators compacts the code but makes it harder to read and debug, it's worth having extra whitespace to make this easier to read.

Reply:
methods are separated and deadband is in robot class